app/Livewire/GlobalSearch.php

```
@@ -0,0 +1,371 @@
+        $this->searchQuery = '';
+        $this->searchResults = [];
+    }
+
+    public static function getCacheKey($teamId)
+    {
+        return 'global_search_items_'.$teamId;
+    }
+
+    public static function clearTeamCache($teamId)
+    {
+        Cache::forget(self::getCacheKey($teamId));
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Cache' in method 'clearTeamCache'. (undefined)

(StaticAccess)